src/prefect/engine/cloud/flow_runner.py

```diff
@@ -182,9 +182,9 @@ def interrupt_if_cancelling() -> None:
                             "Error getting flow run info", exc_info=True
                         )
                         continue
-                    else:
-                        self.logger.debug(
-                            "Successfully queried server for flow run state: %r",
```

Comment:
Sorry, I added this when debugging the cancellation implementation. Definitely no longer needed.

Reply:
Ahhhh haha I'll admit I wondered where it came from - I actually do think that "unexpected" states are worth logging so 👍